addons/mail/tests/common.py

```diff
@@ -53,17 +55,16 @@ def mock_mail_gateway(self, mail_unlink_sent=False):
         self._init_mail_mock()
 
         def _ir_mail_server_build_email(model, *args, **kwargs):
-            self._mails.append(kwargs)
-            self._mails_args.append(args)
+            origin._mails.append(kwargs)
```

Comment:
We don't use mail_args anywhere ... but it make me think this could easily be broken is we call using args instead of kwargs indeed. Depends on the calls I suppose. Will check a bit tiens.